Materials/DynamicMaterialTest/EmissiveMaterial.azsl

```diff
@@ -93,9 +93,6 @@ ForwardPassOutput MainPS(VSOutput IN)
     lightingData.tileIterator.Init(IN.m_position, PassSrg::m_lightListRemapped, PassSrg::m_tileLightData);
     lightingData.Init(surface.position, surface.normal, surface.roughnessLinear);
 
-    // Shadow, Occlusion
-    lightingData.shadowCoords = IN.m_shadowCoords;
```

Comment:
So the new technique doesn't use shadow coords anymore? Can we remove it from the VSOutput struct?

Reply:
Yep, when I to my optimization pass on our forward shaders I'll strip away everything that isn't needed.